src/Utility/ResponseTools.cs

```diff
@@ -0,0 +1,282 @@
+using System;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using OpenAI.Agents;
+using OpenAI.Embeddings;
+using OpenAI.Responses;
+
+namespace OpenAI.Responses;
+
+/// <summary>
+/// Provides functionality to manage and execute OpenAI function tools for responses.
+/// </summary>
+//[Experimental("OPENAIMCP001")
+public class CallLocalAsync

```

Comment:
why public?

Reply:
That was an accidentla rename. It should be `ResponseTools`